lib/3scale/backend/server/falcon.rb

```diff
@@ -0,0 +1,48 @@
+module ThreeScale
+  module Backend
+    module Server
+      class Falcon
+        extend ThreeScale::Backend::Server::Utils
+
+        def self.start(global_options, options, args)
+          # Falcon does not support:
+          # - options[:daemonize]
+          # - options[:logfile]
+          # - options[:errorfile]
+          # - options[:pidfile]
```

Comment:
We can emulate all of those. A potential TODO for the future, likely in common code elsewhere.

Reply:
Yes, I'll leave those as future improvements for now, because they're not needed for the kind of testing we want to do know (#149)